{
  expectedException.expect(InvocationTargetException.class);
  expectedException.expectCause(hasType(UnsupportedOperationException.class).andMessage("UserSession must not be used from within the Compute Engine"));
}
